### PR TITLE
feat(ingest): Add option to change name of database for postgres

### PR DESCRIPTION
**Pull request by aseembansal-gogo**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by hsheth2 — APPROVED** [review]
LGTM

In the future, we'll probably want a more principled way of handling this sort of modification, and ideally would have transformers to handle this. As such, I'm thinking about these changes are more of a stopgap measure.

File: metadata-ingestion/README.md [hunk 1]
```diff
@@ -322,6 +323,7 @@ source:
     password: pass
     host_port: localhost:5432
     database: DemoDatabase
+    database_identifer: DatabaseNameToBeIngested
```
Comment (shirshanka) [inline metadata-ingestion/README.md:326]: Name this field as `database_alias` ?

**Review by shirshanka — CHANGES_REQUESTED** [review]
Suggest we name the field as `database_alias` instead of `database_identifier`

File: metadata-ingestion/README.md (hunk 1 above)
Comment (aseembansal-gogo) [reply metadata-ingestion/README.md]: Good suggestion. That is more appropriate. Done.

**Review by shirshanka — APPROVED** [review]
LGTM!

**Comment by aseembansal-gogo** [comment]
@shirshanka I am not sure where to add the missing dependency for the test. Can you please suggest which file needs this change?

**Comment by shirshanka** [comment]
@aseembansal-gogo : this would be in setup.py

Add `postgres` to the `base_dev_requirements` variable. 
near: 
``` 
      *list(
         dependency
            for plugin in [
                    "bigquery",
                    "bigquery-usage",
                    "looker",
                    "glue",
                    "oracle",
                    "sagemaker",
```

**Pull request merged**